Bender.yml

```diff
@@ -22,7 +22,8 @@ dependencies:
   clint:                    { git: "https://github.com/pulp-platform/clint.git",                  version: 0.2.0  }
   common_cells:             { git: "https://github.com/pulp-platform/common_cells.git",           version: 1.33.0 }
   common_verification:      { git: "https://github.com/pulp-platform/common_verification.git",    version: 0.2.0  }
-  cva6:                     { git: "https://github.com/pulp-platform/cva6.git",                   rev: pulp-v1.0.0 }
+  cva6:                     { git: "https://github.com/pulp-platform/cva6.git",                   rev: pulp-v1    }
+  ara:                      { git: "https://github.com/pulp-platform/ara.git",                    rev: mp/cva6-pulpv1/rebase }
```

Comment:
This should also be merged and get a release before merging this PR.

Reply:
I have released a [second PR](https://github.com/pulp-platform/cheshire/pull/160) that will have tagged commits on `main` branches.